src/index.html

```diff
@@ -1,15 +1,115 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <script type="text/javascript" src="./main.js"></script>
+    <form onsubmit="onSubmit()"
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    autocomplete="disabled"
+    >
```

Comment:
некорректно отформатированы атрибуты
1. каждый атрибут на новой строке
2. атрибут должен находится в 2 отступах от тега
```suggestion
    <form
      onsubmit="onSubmit()"
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      autocomplete="disabled"
    >
```

Reply:
fixed